testplan/runnable/interactive/http.py

```diff
@@ -49,162 +313,34 @@ class TestRunnerHTTPHandler(Entity):
     Also inherits all
     :py:class:`~testplan.common.entity.base.Entity` options.
     """
+
     CONFIG = TestRunnerHTTPHandlerConfig
 
     def __init__(self, **options):
         super(TestRunnerHTTPHandler, self).__init__(**options)
-        self._ip = None
-        self._port = None
+        self._server = None
+        self.task_pool = None
```

Comment:
self.pool? I didn't see self.task_pool be used but self.pool defined in `def run`

Reply:
Woops, yes I changed the name from task_pool -> pool but forgot to update the __init__ declaration.